makefile

```diff
@@ -362,7 +363,7 @@ MFEM_CPPFLAGS  ?= $(CPPFLAGS)
 MFEM_CXXFLAGS  ?= $(CXXFLAGS)
 MFEM_TPLFLAGS  ?= $(INCFLAGS)
 MFEM_INCFLAGS  ?= -I@MFEM_INC_DIR@ @MFEM_TPLFLAGS@
-MFEM_PICFLAG   ?= $(if $(shared),$(PICFLAG))
+MFEM_PICFLAG   ?= $(if $(or $(shared),$(jit)),$(PICFLAG))
```

Comment:
JIT now requires shared, right? If so, we can just check for shared and remove the `jit` variable.

Reply:
`done`